README.md

```diff
@@ -293,6 +293,11 @@ export class Bus extends Car {
 }
 ```
 
+### Container clearing

```

Comment:
Not sure if we should document this function... I doubt someone ever will use this method and if they don't and we need this mostly for our internal usage don't think we should document this by wasting users minds